README_CODE

```diff
@@ -176,7 +176,7 @@ Note that the default controller is "observer" and the "id" parameter is
 shortened to a bare integer:
 
   http://mo.org/42                      Show observation #42.
-  http://mo.org/observer/show_user/252  Show summary for user #3.
```

Comment:
Would you mind correcting it from 252 to 3 (or vice versa)?  Trivial, but unnecessarily confusing.

Reply:
Updated